src/server_manager/messages/am.json

```diff
@@ -1,892 +1,196 @@
-        "content": "{closeLink}"
-      },
-      "START_OF_LINK": {
-        "content": "{openLink}"
-      }
-    }
-  },
-  "share_invite": {
-    "message": "የኔን አውትላይን አገልጋይ መጋራት እንዲችሉ ጋብዤዎታለሁ። የትም ሆነው ቢሆን በይነመረብ እንዳሻዎ ሊጠቀሙበት ይችላሉ። የአውትላይን መተግበሪያውን ለማውረድ እና ከበይነመረብ ለመገናኘት የሚከተሉትን ትዕዛዛት ይከተሉ። ",
-    "description": "This string appears as a paragraph within a dialog that provides an invitation to share access to a server. Belongs to the instructions for the user with whom access is being shared."
-  },
-  "share_invite_copied": {
-    "message": "ቅጂ ተይዟል",
-    "description": "This string appears as a label within a dialog that provides an invitation to share access to a server. Displayed when the user clicks a button to copy the invitation text to the clipboard."
-  },
-  "share_invite_copy": {
-    "message": "ግብዣውን ይቅዱ",
-    "description": "This string appears as a button within a dialog that provides an invitation to share access to a server. Clicking it copies the invitation instructions text to the clipboard."
-  },
-  "share_invite_copy_access_key": {
-    "message": "የመግቢያ ቁልፉን ይቅዱ",
-    "description": "This string appears as a paragraph within a dialog that provides an invitation to share access to a server. Belongs to the instructions for the user with whom access is being shared."
-  },
-  "share_invite_instructions": {
-    "message": "ጊትኸብ ላይ ያለውን የግብዣ ትዕዛዞች ይከተሉ",
-    "description": "This string appears as a list item within a dialog that provides an invitation to share access to a server. Belongs to the instructions for the user with whom access is being shared."
-  },
-  "share_invite_trouble": {
-    "message": "የግብዛ መገናኛውን ለማግኘት ችግር ገጠምዎት?",
-    "description": "This string appears as a paragraph within a dialog that provides an invitation to share access to a server. Belongs to the instructions for the user with whom access is being shared."
-  },
-  "share_title": {
-    "message": "መግቢያዎን ያጋሩ",
-    "description": "This string appears as the header of a dialog that provides an invitation to share access to a server."
-  },
-  "terms_of_service": {
-    "message": "1 የ$START_OF_LINK$ አውትላይን መመሪያዎችን አንብቤ ተረድቻለሁ።\n2$END_OF_LINK$",
-    "description": "This string appears within an overlay the first time the user runs the app. Prompts the user to review and accept Outline's terms of service.",
-    "placeholders": {
-      "END_OF_LINK": {
-        "content": "{closeLink}"
-      },
-      "START_OF_LINK": {
-        "content": "{openLink}"
-      }
-    }
-  }
+  "about-outline": "Outline (አውትላይን) በ <a href={jigsawUrl}>Jigsaw</a> የተዘጋጀ ክፍት ምንጭ ፕሮጀክት ሲሆን  ለዜና አቅራቢ ድርጅቶችና ጋዜጠኞች ደኅንነቱ የተጠበቀ የኢንተርኔት አጠቃቀምን እንዲያገኙ ታልሞ የተዘጋጀ ነው፡፡ <br><br> Outline (አውትላይን) በ  <a href={shadowsocksUrl}>Shadowsocks</a> የተደገፈ አዲስ ምርት ነው፡፡  በ <a href={gitHubUrl}>GitHub</a> ላይ ለፕሮግራሙ አስተዋፅዖ ማድረግ ይችላሉ፡፡ በተጨማሪም <a href={redditUrl}>Reddit</a> እና <a href={mediumUrl}>Medium</a> ላይ ሊከታተሉን እና መድረካችንን እና አገልግሎቶቻችንን ስናሳድግ መስማት ይችላሉ።  ",
```

Comment:
There's an extra space here before the <a> tag compared to the original

Reply:
These were generated from reviewed translations, so I think it should be fine.